### ISSUE TITLE
Plans for Gradle?

### ISSUE THREAD

**Issue by vxcamiloxv** [jdee-emacs/jdee#115]
What plans have JDEE fro support Gradle?

**Comment by oldButNotWise** [comment]
I don't know what you expect, but this project works for me. It's not polished, and should be considered "alpha" quality.

https://github.com/oldButNotWise/jdee-gradle

**Comment by vxcamiloxv** [comment]
In this moment I have a jdee.gradle for get .jars and .class similar to the .gradle in jdee-gradle project but will be interesting if jdee get the necesary for gradle project without jdee.gradle by project, more automatic.

**Comment by oldButNotWise** [comment]
The jdee-gradle code I referenced defines a gradle target that generates a JDEE project file that sets the various source and class paths, based on info available to gradle. You do have to run the target by hand, but I find it's better than nothing.

**Comment by vxcamiloxv** [comment]
Yes it's better than nothing, but the jdee.gradle in jdee-gradle not work with android gradle project or last gradle versión.
